### ISSUE TITLE
Create a process for versioning Templates 

### ISSUE THREAD

**Issue by TCMeldrum** [UserOfficeProject/user-office-project-issue-tracker#832]
If a template changes an old proposal using that template could lose the link to some answers in the database and therefore will never be displayed. When we need to make large changes to the template we should clone the template and make the changes to the clone so old proposals are not affected by the changes. This all can be done through the interface currently so would require no code changes but we should formalize the process such as what constitutes a breaking change in the template. This should probably also be done in conjunction with the user office.

**Comment by TCMeldrum** [comment]
Blocked by #834

**Comment by simonfernandes** [comment]
Backlog pruning notes:

Not needed for now.
